dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -282,6 +282,7 @@ export default defineComponent({
      * @param document the certificate as combined object of reference id and file content
      */
     updateDocumentList(fieldName: string, document: DocumentToUpload) {
+      console.log("createLksgDataset", { fieldName, document });
```

Comment:
Ok.  But the broken document uploader has nothing to do with the renaming btw.
It was broken before already (due to the "multiple" prop of the FileUploader set to "true")

Reply:
I think it does, because the marked element in the screenshot is no longer shown and no reports a saved in the dataset.
![image](https://github.com/d-fine/Dataland/assets/104447806/796334c1-51db-4245-9be9-bb1f14838d61)

Reply:
Ok but then there are actually two bugs.
Because the one I mentioned (multiple selection possible on one certificate upload) is also an issue.

Reply:
Should be fixed now. I've tested it locally against all frameworks.
Pushed.

Reply:
👍